src/browser/browser.ts

```diff
@@ -168,9 +168,11 @@ export class Browser {
       });
     }
 
-    if (options.headers && Object.keys(options.headers).length > 0) {
-      this.log.debug(`Setting extra HTTP headers for page`, 'headers', options.headers);
-      await page.setExtraHTTPHeaders(options.headers as any);
+    if (options.headers && options.headers['Accept-Language']) {
+      const headers = { 'Accept-Language': options.headers['Accept-Language'] };
+      this.log.debug(`Setting extra HTTP headers for page`, 'headers', headers);
```

Comment:
do we want to keep these debug logs for future troubleshooting?

Reply:
I didn't add this one but yes I think it can be interesting for troubleshooting.